src/metabase/api/user.clj

```diff
@@ -177,27 +198,42 @@
   Takes `query` for filtering on first name, last name, email.
   Also takes `group_id`, which filters on group id."
   [status query group_id include_deactivated]
-  {status                 (s/maybe s/Str)
-   query                  (s/maybe s/Str)
-   group_id               (s/maybe su/IntGreaterThanZero)
-   include_deactivated    (s/maybe su/BooleanString)}
-  (when (or status include_deactivated)
-    (validation/check-group-manager))
+    {status              (s/maybe s/Str)
+     query               (s/maybe s/Str)
+     group_id            (s/maybe su/IntGreaterThanZero)
+     include_deactivated (s/maybe su/BooleanString)}
+  (validation/check-group-manager)
```

Comment:
Think we can skip this generic check if `group_id` is provided, since `check-manager-of-group` is already checking whether the user is a group manager. (But as I mentioned above, it would be nice if all this behavior could be rolled into one function)